docs/user/openstack/README.md

```diff
@@ -105,63 +102,106 @@ openstack network list --long -c ID -c Name -c "Router Type"
 
-* https://github.com/shiftstack-dev-tools/ocp-doit
+[ocp-doit]: https://github.com/shiftstack-dev-tools/ocp-doit
 
-## API Access
+## Initial Setup
 
-All the OpenShift nodes are created in an OpenStack tenant network and as such, can't be accessed directly in most openstack deployments. The installer does not create any floating IP addresses, but does need access to the OpenShift's API as it is being deployed. We will briefly explain how to set up access to the openshift api with and without floating IP addresses.
+If you don't already have the OpenShift Client Tools, follow [these steps](https://docs.openshift.com/enterprise/3.0/cli_reference/get_started_cli.html). Once you have that set up, download the latest stable versions of the [installer](https://mirror.openshift.com/pub/openshift-v4/clients/ocp/latest/). You will need to unzip the tarball with the following command:
 
-### Using Floating IPs
+```sh
+tar -xvf openshift-install-OS-VERSION.tar.gz
+```
 
-This method allows you to attach two floating IP addresses to endpoints in OpenShift.
+For the time being, the OpenStack platform is not supported in the prompts. We will instead provide you with a template `install-config.yaml` to use below. If you want to go beyond the basic customizations we display below, there is more information available in [this document](customization.md). We have labled sections in the yaml below with the following syntax: (#). Look for the numbers below the template for explanations of how to complete those sections.
 
-First, create a floating IP address for the API:
+```yaml
+apiVersion: v1
+baseDomain: (1)
+clusterID:  (2)
+controlPlane:
+  name: master
+  platform: {}
+  replicas: (3)
+compute:
+  name: worker
+  platform:
+    openstack:
+      type: (4)
+  replicas: (5)
+metadata:
+  name: (6)
+networking:
+  clusterNetworks:
+  - cidr: (7)
+    hostSubnetLength: 9
+  type: OpenShiftSDN
+platform:
+  openstack:
+    region: (8)
+    cloud:  (9)
+    externalNetwork: (10)
+    computeFlavor: (11)
+    lbFloatingIP: (12)
+pullSecret: (13)
+sshKey: (14)
+```
 
-    $ openstack floating ip create <external network>
+1. The domain name and top level domain you want the api to host services at. Example: `example.com`.
+2. A unique hash or identifier for this particular cluster.
+3. The number or master nodes to deploy. See the [OpenStack Requirements](#openstack-requirements) for details on requirements
+4. The Flavor to use for worker nodes. See the [OpenStack Requirements](#openstack-requirements) for details on requirements
+5. The number of worker nodes to deploy. See the [OpenStack Requirements](#openstack-requirements) for details on requirements
+6. A name that identifies your cluster. Your api will be hosted at `<cluster name>.<base domain>`, and its resources will be prepended by `<cluster name>-<clusterID>-`
```

Comment:
Correct me if I'm wrong but I think that `clusterID` is generated and is not the same one as in 2). If that's the case, we should make it clear.